src/FlatSharp/TypeModel/UnityTypeModelProvider.cs

```diff
@@ -0,0 +1,38 @@
+﻿/*
+ * Copyright 2022 Unity Technologies
```

Comment:
I opened another comment about this in the past, but just so you know, I'm going to change all copyright headers in the entire repo to read "Copyright 2022 by the FlatSharp contributors" after you commit your changes. I'm happy to add Unity Technologies to an official contributor list as well -- please let me know if you have a preference.

Reply:
https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license